tests/unit/cli/admin/test_metadata.py

```diff
@@ -1194,3 +1194,19 @@ def test_metadata_sign_delete(self, client, test_context):
                 metadata.Methods.GET,
             ),
         ]
+        assert metadata.send_payload.calls == [
+            pretend.call(
+                test_context["settings"],
+                URL.METADATA_SIGN_DELETE.value,
+                {"role": "root"},
+                "Metadata sign delete accepted.",
+                "Metadata delete sign",
+            )
+        ]
+        assert metadata.task_status.calls == [
+            pretend.call(
+                "task_id",
+                test_context["settings"],
+                "Metadata delete sign",
+            )
+        ]
```

Comment:
@ivanayov please make sure that on each function you use `pretend` you also check its calls as I did here.